configs/openimages/README.md

```diff
@@ -1,6 +1,9 @@
 # Open Images Dataset
-<!-- [DATASET] -->
 
+> [Open Images Dataset](https://arxiv.org/abs/1811.00982)
+
+<!-- [DATASET] -->
+<!-- [ALGORITHM] -->
```

Comment:
We should assign one label to this config.

Reply:
Just use Dataset is fine